src/Raven.Server/Utils/Cli/RavenCli.cs

```diff
@@ -401,17 +402,28 @@ private static bool CommandTimer(List<string> args, RavenCli cli)
 
         private static bool CommandLog(List<string> args, RavenCli cli)
         {
+            var withConsole = !(args.Count == 2 && args[1].Equals("no-console"));
+                        
             switch (args.First())
             {
                 case "on":
-                    LoggingSource.Instance.EnableConsoleLogging();
+                case "information":
+                    if (withConsole)
+                        LoggingSource.Instance.EnableConsoleLogging();
                     LoggingSource.Instance.SetupLogMode(LogMode.Information, cli._server.Configuration.Logs.Path.FullPath);
-                    WriteText("Logging set to ON", ConsoleColor.Green, cli);
+                    WriteText("Logging set to ON (information)", ConsoleColor.Green, cli);
                     break;
                 case "off":
+                case "none":
                     LoggingSource.Instance.DisableConsoleLogging();
                     LoggingSource.Instance.SetupLogMode(LogMode.None, cli._server.Configuration.Logs.Path.FullPath);
-                    WriteText("Logging set to OFF", ConsoleColor.DarkGreen, cli);
+                    WriteText("Logging set to OFF (none)", ConsoleColor.DarkGreen, cli);
+                    break;
+                case "operations":
+                    if (withConsole)
+                        LoggingSource.Instance.EnableConsoleLogging();
+                    LoggingSource.Instance.SetupLogMode(LogMode.None, cli._server.Configuration.Logs.Path.FullPath);
+                    WriteText("Logging set to 'operations'", ConsoleColor.DarkGreen, cli);
```

Comment:
Shouldn't be: "Logging set to ON (operations)"
to be consistent with other options?